### PR TITLE
Aggressive cows added

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

Aggressive cows code added in cp(competitive programming) folder added.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#3215

### CODE REVIEW

**Comment by gayatri517** [comment]
@ankitaggarwal23 Kindly review my PR. 
Thank you.

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@
 - [Painting Fence Algorithm](cp/Paint_Fence.cpp)
 - [All String permutation by using recursion](cp/recursion_string_permutation.cpp)
 - [Peak element](cp/Peak_element.cpp)
+- [Aggressive cows added](cp/aggressive_cows.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Remove the word "added"

File: C-Plus-Plus/cp/aggressive_cows.cpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		{
+			last_cow = stalls[i];
+			cnt++;
+			if (cnt == c)
+			{
+				return true;
+			}
+		}
+	}
+
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/aggressive_cows.cpp:27]: Remove the extra line

File: C-Plus-Plus/cp/aggressive_cows.cpp [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	return false;
+}
+
+int main()
+{
+	int stalls[100], n, cows;
+	//User input : 3
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/aggressive_cows.cpp]: Remove the redundant comments. We said use cout to display proper message like for example cout<<" Enter the value of n and Cows; etc. like that!

File: C-Plus-Plus/cp/aggressive_cows.cpp [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	cin >> n >> cows;
+
+	//User input : 1 2 4 8 9
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/aggressive_cows.cpp]: Remove the redundant comments. We said use cout to display proper message.

File: C-Plus-Plus/cp/aggressive_cows.cpp [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+	for (int i = 0; i < n; ++i)
+	{
+		cin >> stalls[i];
+	}
+
+	int s = 0;
+	int e = stalls[n - 1] - stalls[0];
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/aggressive_cows.cpp]: Use proper naming convention. Don't use single letter.

File: C-Plus-Plus/cp/aggressive_cows.cpp [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+			ans = mid;
+			s = mid + 1;
+		}
+		else
+		{
+			e = mid - 1;
+		}
+	}
+
+	//Output message : 3
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/aggressive_cows.cpp]: Remove the redundant comments.

File: C-Plus-Plus/cp/aggressive_cows.cpp [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+}
+/* Example :
+1)Input :
+5 3
+1 2 4 8 9
+Output : 3
+
+Time Complexity : O(nlog n)
+Space Complexity : O(n)
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/aggressive_cows.cpp]: Add a new line at the end of file

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@ankitaggarwal23 I have made the changes. Kindly review them. 
Thank you.

**Comment by gayatri517** [comment]
@ankitaggarwal23 kindly review my PR.

**Comment by gayatri517** [comment]
@ankitaggarwal23 Please review my PR.
Thank you.

**Review by rohanbabbar04 — COMMENTED** [review]
Add a new line at EOF
@gayatri517 Merge conflicts do resolve

**Comment by gayatri517** [comment]
@rohanbabbar04 I have added a new line at EOF. Please check it.

**Comment by gayatri517** [comment]
@rohanbabbar04 Kindly check it I have tried resolving the merge conflicts.

**Comment by gayatri517** [comment]
@HarshCasper please take look at my PR.
Thank you.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**